lib/linter.js

```diff
@@ -901,8 +906,15 @@ module.exports = class Linter {
                 options.filename
             );
         } catch (err) {
+            err.message += `\nOccurred while linting ${options.filename}`;
             debug("An error occurred while traversing");
             debug("Filename:", options.filename);
+            if (err.currentNode) {
+                const { line } = err.currentNode.loc.start;
```

Comment:
Can we get column information here too?

Reply:
We could. The reason I didn't include it is because it's even less accurate than the line number - it won't always point exactly to the spot that caused the crash. Even the line number won't always do that, depending on how the rule is implemented.